scripts/get-access-key.sh

```diff
@@ -32,5 +32,5 @@ function getAccessKey() {
                       --data-urlencode "scope=applied-permissions/admin" \
                       --data-urlencode "description=Created_with_script_in_TF_provider" | jq -r .access_token)
 
-  echo ${scoped_access_key}
+  echo "${scoped_access_key}"
```

Comment:
Doesn't this put the double quotes back in the shell when it is echo out?

Reply:
Tested, no double quotes. But this way we prevent globing and word-splitting per [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)